src/actions/collectiblesActions.js

```diff
@@ -46,19 +46,22 @@ export const fetchCollectiblesAction = () => {
         asset_contract: assetContract,
         name,
         description,
-        image_preview_url: image,
+        image_url: fullImage,
+        image_preview_url: preview,
         image_thumbnail_url: thumbnail,
       } = collectible;
       const { name: category, address: contractAddress } = assetContract;
       const collectibleName = name || `${category} ${id}`;
 
+      const image = (/\.(png|gif)$/i).test(fullImage) ? fullImage : '';
```

Comment:
Shall we test `preview` for extensions as well? The problem here is that some images are stored on ipfs and we have no way to display them

Reply:
Oh, okay, we can test that as well. Will update

Reply:
Updated